public/scss/_base.scss

```diff
@@ -93,8 +93,8 @@ main {
     & label {
       border: 1px solid transparent;
       display: block;
-      font-size: 18px;
-      font-weight: bold;
+      font-size: $medium-font;
```

Comment:
Good to start using vara for all of the font-sizes - helps keep things limited to specific sizes.